tools/scripts/generate_binary_build_matrix.py

```diff
@@ -223,9 +223,6 @@ def generate_libtorch_matrix(
 
     ret: List[Dict[str, str]] = []
 
-    if os == "macos-arm64" or os == "macos":
```

Comment:
I think this was a mistake to remove libtorch macos wheels from this matrix. Hence bringing it back